components/raftstore-v2/src/worker/tablet_gc.rs

```diff
@@ -295,6 +315,9 @@ impl<EK: KvEngine> Runner<EK> {
                             "path" => path.display(),
                         )
                     });
+                if success.is_ok() &&let Err(e) = mgr.delete_snapshot(region_id, None){
```

Comment:
Destroying a tablet doesn't mean the snapshot is outdated.

Reply:
removed